salt/master.py

```diff
@@ -1036,6 +1036,11 @@ def _handle_payload(self, payload):
 
         :param dict payload: The payload route to the appropriate handler
         """
+        if payload.get("cmd") == "_auth":
+            if self.opts["master_stats"]:
```

Comment:
Should we use:
```suggestion
            if self.opts.get("master_stats", False):
```

to be on the safe side, as we do in `salt/channel/server.py`?

Reply:
with `master.py` there are another places which are relying on the presence of these parameter in the `opts`, the other master stats related conditions are using `self.opts["master_stats"]`, originally I was using exactly `self.opts.get("master_stats", False)`, but on preparing upstream PR changed it to the original way of checking this parameter.

Reply:
ACK. Thanks for claryfing